### DISCUSSION TITLE
Brainstorming a new name for the bot and project packages

### DISCUSSION THREAD

**Discussion by josefaidt** [aws-amplify/discord-bot#28] [Ideas]
Hello all :wave:

Since the bot's initial release, we thought it would be a great idea to give it a name! Currently the bot is named `amplify-bot` and all project packages use the `@amplify-discord-bots` scope (this repository name). It would be great to align both the bot name and package scope to be consistent. Opening this discussion for all feedback regarding the naming of the Discord bot and supporting packages' scope 🙂

**Comment by renegoretzka** [comment]
Here are a few suggestions I like.

- [ ] HeyAmplify
- [ ] AmplifyOne
- [ ] AmplifyBeast
- [ ] AmplifyBuddy
- [ ] AmplifyZen
- [ ] AgentAmplify

**Reply by josefaidt** [reply]
I like HeyAmplify!!

**Reply by renegoretzka** [reply]
Yeah, I think that's also my favourite out of these 🥇

**Comment by n123456789k** [comment]
JeffBezos
